icu4j/main/tests/core/src/com/ibm/icu/dev/test/number/NumberFormatterApiTest.java

```diff
@@ -3392,6 +3515,27 @@ public void fieldPositionCoverage() {
                     expectedFieldPositions);
         }
 
+        {
+            String message = "Measure unit field position with prefix and suffix, built-in m/s";
+            FormattedNumber result = assertFormatSingle(
+                    message,
+                    "measure-unit/speed-meter-per-second unit-width-full-name",
+                    "unit/meter-per-second unit-width-full-name",
+                    NumberFormatter.with().unit(MeasureUnit.METER_PER_SECOND).unitWidth(UnitWidth.FULL_NAME),
```

Comment:
With this now so closely matching the "composed m/s" example above, I probably don't need both anymore and could collapse this into one again?